composer.json

```
@@ -24,8 +24,8 @@
     "require": {
         "php": "^8.0",
         "astrogoat/strata": "^0.3.0|^0.4.0|^0.5.0|^0.6.0",
```

Latest version of Strata is 0.7.3.

```suggestion
        "astrogoat/strata": "^0.7.3",
```